agent/grpc-external/services/peerstream/stream_test.go

```diff
@@ -868,8 +869,8 @@ func TestStreamResources_Server_CARootUpdates(t *testing.T) {
 	})
 }
 
-// Test that when the client doesn't send a heartbeat in time, the stream is terminated.
-func TestStreamResources_Server_TerminatesOnHeartbeatTimeout(t *testing.T) {
+// Test that when the client doesn't send a heartbeat in time, the stream is disconnected.
+func TestStreamResources_Server_DisconnectsOnHeartbeatTimeout(t *testing.T) {
```

Comment:
renamed this to disambiguate between Terminated messages and disconnects on errors